src/app/lib/config/services/uzbek.ts

```diff
@@ -32,337 +32,650 @@ const baseServiceConfig = {
+      },
+      socialEmbed: {
+        caption: {
+          textPrefixVisuallyHidden: 'Видео тагсўзи, ',
+          text: 'Огоҳлантириш:Учинчи манба материалида реклама бўлиши мумкин',
+          articleText:
+            'Огоҳлантириш: Би-би-си ташқи саҳифалардаги контент учун масъул эмас',
+          articleAdditionalText:
+            '%provider_name% бу контентда реклама бўлиши мумкин',
+        },
+        fallback: {
+          text: 'Бу материалга кириш имконсиз',
+          linkText: 'Кўпроқ кўринг %provider_name%',
+          linkTextSuffixVisuallyHidden: ', ташқи',
+          warningText: 'Би-би-си ташқи сайтлар мазмуни учун масъул эмас.',
+        },
+        skipLink: {
+          text: 'Ўтказиб юборинг %provider_name% пост ',
+          endTextVisuallyHidden: 'Охири %provider_name% пост',
+        },
+        consentBanner: {
+          heading: `[social_media_site] контентига рухсат бериш`,
+          body: `Айни мақолада [social_media_site] томонидан тақдим қилинган контент мавжуд. Биз бу контент юкланмасидан аввал сизнинг розилигингизни сўраймиз, чунки улар куки ва бошқа технологиялардан фойдаланган бўлиши мумкин. Сиз   [social_media_site] [link] ҳаволасида кукиларга доир  [/link] ва шахсий маълумотларга  оид қоидалар ҳақида аввал ўқиб,  кейин қабул қилишга рози бўлишингиз  мумкин. Кўриш учун “қабул қилиш ва давом этиш”ни танланг.`,
+          button: 'Қабул қилинг ва давом этинг',
+        },
+      },
+      include: {
+        errorMessage:
+          'Sorry, we can’t display this part of the story on this lightweight mobile page.',
+        linkText: 'View the full version of the page to see all the content.',
+      },
+      topStoriesTitle: 'Бош мақола',
+      featuresAnalysisTitle: 'Муҳаррир танлови',
+      latestMediaTitle: 'Сўнгги',
+    },
+    mostRead: {
+      header: 'Энг кўп ўқилган',
+      lastUpdated: 'Сўнгги янгиланиш:',
+      numberOfItems: 5,
+      hasMostRead: true,
+    },
+    footer: {
+      trustProjectLink: {
+        href: 'https://www.bbc.com/uzbek/institutional-50220995',
+        text: 'Нега сиз Би-би-сига ишонишингиз мумкин?',
+      },
+      externalLink: {
+        href: 'https://www.bbc.co.uk/editorialguidelines/guidance/feeds-and-links',
+        text: 'Ташқи линкларга бизнинг ёндашувимиз қандайлиги ҳақида маълумотга эга бўлинг',
+      },
+      links: [
+        {
+          href: 'https://www.bbc.com/uzbek/institutional-36824297',
+          text: 'Фойдаланиш шартлари',
+        },
+        {
+          href: 'https://www.bbc.com/uzbek/institutional-36824300',
+          text: 'Шахсий ҳаёт махфийлиги сиёсати',
+        },
+        {
+          href: 'https://www.bbc.com/usingthebbc/cookies/',
+          text: 'Куки(Cookies)',
+        },
+        {
+          href: 'https://www.bbc.co.uk/uzbek/send/u50853929',
+          text: "'Bi-bi-si bilan bog’laning'",
+        },
+        {
+          id: 'COOKIE_SETTINGS',
+          href: '#',
+          text: 'Do not share or sell my info',
```

Comment:
same here?

Reply:
Same as above, about half of services have this in English, but can ask.